changelog.d/377.misc

```diff
@@ -0,0 +1 @@
+Fix email templates to be valid python format strings.
```

Comment:
Should this have been a bugfix?

Reply:
I presumed the files were to be treated as sample templates rather than the canonical ones, so didn't consider a broken example to be a "bug"

Reply:
Ah, fair. I'm not sure if we're really clear at how one is to deploy sydent since... it isn't widely deployed beyond us. Carry on! 👍

Reply:
> it isn't widely deployed beyond us

For what it's worth, I think this is increasingly untrue - *despite* the fact that the deployment process is undocumented and fraught with footguns (https://github.com/matrix-org/sydent/issues/131 is particularly relevant here)